source/Examples/Resources/CMDistributionPoint/CMDistributionPoint_Present.ps1

```diff
@@ -0,0 +1,32 @@
+<#
+    .SYNOPSIS
+        A DSC configuration script to add a distribution point to Configuration Manager.
+#>
+Configuration Example
+{
+    Import-DscResource -ModuleName ConfigMgrCBDsc
+
+    Node localhost
+    {
+        CMDistributionPoint ExampleSettings
+        {
+            SiteCode                        = 'Lab'
+            SiteServerName                  = 'DP01.contoso.com'
+            Description                     = 'Primary distribution point'
+            MinimumFreeSpaceMB              = 100
+            PrimaryContentLibraryLocation   = 'F'
+            SecondaryContentLibraryLocation = 'C'
+            PrimaryPackageShareLocation     = 'F'
+            SecondaryPackageShareLocation   = 'C'
+            CertificateExpirationTimeUtc    = '5/26/2025'
+            ClientCommunicationType         = 'Https'
+            BoundaryGroups                  = @('TestBoundary1','TestBoundary2')
+            BoundaryGroupStatus             = 'Add'
```

Comment:
I tested this PR and everything seems to be working except adding boundary groups. If the boundary group to add doesn't exist, it goes to the BoundaryGroupAbsent throw.